backend/users/controllers/leaderboardController.js

```diff
@@ -15,6 +15,7 @@ const getLeaderboard = async (req, res) => {
       const platform = req.query.platform.toLowerCase();
       users = await User.find({
         [`${platform}.rating`]: { $exists: true, $ne: null },
+        digitomize_rating: { $gt: 0 },
```

Comment:
This is not really required, this part is under `if (req.query.platform)` so for platforms we dont need to check digitomize rating.